huly.nginx

```diff
@@ -19,6 +19,19 @@ server {
         proxy_pass http://account:3000/;
     }
 
+    location /_love {
```

Comment:
There is no `love` container, so nginx won't start.

Reply:
Nginx will start; it does not check if everything is reachable.
This is added incase a user wants to add the love container themselves with their own livekit instance.
I will add an option to selfhost livekit in another pr.
I do see now that I should update the port.